### PR TITLE
Add display flows for listing details images

### PR DESCRIPTION
**Pull request by henryhu1**
~I have no idea if what I'm doing is correct or even makes sense but the~ behaviour looks good to me

### CODE REVIEW

**Review by andr-dev — APPROVED** [review]
Looks about right, I'll let Min-Kyu take a look.

File: android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt [hunk 1]
```diff
@@ -411,6 +424,20 @@ fun ListingDetailsPageView(
 	}
 }
 
+@Composable
+fun ImageBoxPlaceholder(
+	modifier: Modifier = Modifier,
+	contentAlignment: Alignment = Alignment.Center,
+	content: @Composable () -> Unit)
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt]: It should be `) {` if you know what I mean.

File: android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageViewModel.kt [hunk 2]
```diff
@@ -51,12 +53,17 @@ class ListingDetailsPageViewModel @Inject constructor(
...
 	private val imagesStream: Observable<List<Bitmap>> = imageIdsStream.map {
+		isFetchingImagesStream.onNext(true)
 		runBlocking {
 			it.map { id ->
 				async {
 					runCatching {
 						listingsApi.listingsImagesGet(id)
-					}.getOrNull()
+					}.getOrElse {
+						val currentErrors = fetchingImagesErrorsStream.value ?: emptyList()
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageViewModel.kt]: ~~Why are we growing the list of errors?~~
~~Shouldn't the most recent set of errors be sufficient?~~

~~Also, in general, we aren't displaying the specific errors to the user right? We're just telling them that we had an error or some form.~~
~~I don't think we need to keep the `Throwables` around.~~

**Review by leeminkyu16 — COMMENTED** [review]
Some questions about stuff

File: android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt (hunk 1 above)
Comment (henryhu1) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt]: oops my bad, yes

File: android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageViewModel.kt (hunk 2 above)
Comment (henryhu1) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageViewModel.kt]: yeah there's no UI error handling per what went wrong
I thought having all the errors would be useful later on but in that case a boolean would be enough to indicate an error occurred, wouldn't it?

Comment (leeminkyu16) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageViewModel.kt]: Let me think about this for a sec.

Comment (henryhu1) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageViewModel.kt]: removed this `Throwable` tracking stuff

**Review by leeminkyu16 — APPROVED** [review]
Where error stuff go?
Anyways, let's gamble, try merging.

**Comment by henryhu1** [comment]
> Where error stuff go? Anyways, let's gamble, try merging.

At this point I don't think it's necessary, the picture display issue is fixed so if needed we can go back to error handling

**Review by leeminkyu16 — APPROVED** [review]
Let's Gamble, Try Merging

**Pull request merged**